Tests/WeaverTests/RealmWeaver.Tests/WeaverTests.cs

```diff
@@ -29,12 +29,12 @@
 using NUnit.Framework;
 using Realms;
 using System.ComponentModel;
-using AssemblyToProcess;
 using Mono.Cecil.Cil;
 
 namespace RealmWeaver
 {
     [TestFixture(WeaverOptions.RealmOnlyNoPropertyChanged)]
+    [TestFixture(WeaverOptions.PCLRealmOnlyNoPropertyChanged)]
```

Comment:
Can we somehow generate a matrix with all the possible combinations of PropertyChanged and PCL?